src/JointAxis.cc

```diff
@@ -121,35 +122,35 @@ Errors JointAxis::Load(ElementPtr _sdf)
   // Load dynamic values, if present
   if (_sdf->HasElement("dynamics"))
   {
-    sdf::ElementPtr dynElement = _sdf->GetElement("dynamics");
-
-    this->dataPtr->damping = dynElement->Get<double>("damping",
-        this->dataPtr->damping).first;
-    this->dataPtr->friction = dynElement->Get<double>("friction",
-        this->dataPtr->friction).first;
-    this->dataPtr->springReference = dynElement->Get<double>("spring_reference",
-        this->dataPtr->springReference).first;
-    this->dataPtr->springStiffness = dynElement->Get<double>("spring_stiffness",
-        this->dataPtr->springStiffness).first;
+    sdf::ElementPtr dynElement = _sdf->GetElement("dynamics", errors);
+
+    this->dataPtr->damping = dynElement->Get<double>(errors,
+        "damping", this->dataPtr->damping).first;
+    this->dataPtr->friction = dynElement->Get<double>(errors,
+        "friction", this->dataPtr->friction).first;
+    this->dataPtr->springReference = dynElement->Get<double>(errors,
+        "spring_reference", this->dataPtr->springReference).first;
+    this->dataPtr->springStiffness = dynElement->Get<double>(errors,
+        "spring_stiffness", this->dataPtr->springStiffness).first;
   }
 
   // Load limit values
   if (_sdf->HasElement("limit"))
   {
     sdf::ElementPtr limitElement = _sdf->GetElement("limit");
```

Comment:
I believe this line might have missed out passing in `errors`

Reply:
Yup, thanks for the catch, added.